_includes/head.html

```diff
@@ -1,11 +1,21 @@
 <head>
+
+    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <link rel="icon" type="image/png" href="./img/logo/logoOA_svg.png">
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-
-    <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
+    <meta name="keywords" content="foss, open source, organisations, astronomy,
+    OpenAstronomy, astrophysics, GSoC, projects, members, {{ site.members | join: ', ' }}" />
```

Comment:
@nabobalis I actually included all possible keywords which can be possibly included in the keywords. Since OpenAstronomy has projects from different members and organisations I though to include them in the keywords. If you say then I can exclude them.